.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended"
+    ],
+    "overrides": [
```

Comment:
There should be an additional override section here for Jest files. To complete the CA we expect both Jest and Cypress to be provided